nitro_sg.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*"] + ["Rakefile", "README.md"]
 
-  s.add_dependency "rails", ">= 4.2.9", "< 6.0"
+  s.add_dependency "rails", "~> 5.1.6"
```

Comment:
In `nitro-web`, we are exactly here `~> 5.1.6` for `rails`

Reply:
This makes it more difficult to upgrade nitro-web. What is the reason for this?

Reply:
Well we certainly shouldn't be on `4.x` whatsoever. How about `">= 5.1.6", "< 6.0"`? That should be flexible enough right?